test/e2e/tests/metamask-responsive-ui.spec.js

```diff
@@ -164,12 +164,9 @@ describe('Metamask Responsive UI', function () {
 
         // Import Secret Recovery Phrase
         const restoreSeedLink = await driver.findClickableElement(
-          '.unlock-page__link--import',
-        );
-        assert.equal(
-          await restoreSeedLink.getText(),
-          'import using Secret Recovery Phrase',
+          '.unlock-page__link',
         );
+        assert.equal(await restoreSeedLink.getText(), 'Forgot Password?');
```

Comment:
Same here maybe `getText()` isn't case sensitive?

Reply:
Same as above.

Reply:
Got it!